weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx

```diff
@@ -210,101 +344,221 @@ export const HomeExpressionPreviewParts: React.FC<{
           {/* </Unclickable> */}
         </LayoutElements.Block>
       </LayoutElements.VBlock>
-      {generators.loading || refinedExpression.loading || isGenerating ? (
-        <LayoutElements.VStack
-          style={{
-            alignItems: 'center',
-            justifyContent: 'center',
-            gap: '16px',
-            color: '#8e949e',
-          }}>
-          <WeaveAnimatedLoader style={{height: '64px', width: '64px'}} />
-        </LayoutElements.VStack>
-      ) : (
-        generators.result.length > 0 && (
-          <LayoutElements.VBlock style={{gap: '8px'}}>
+      {isLoadingTemplates ? 
+        <Loader /> : (
+          <LayoutElements.VBlock style={{gap: '8px', paddingBottom: '32px'}}>
             <LayoutElements.BlockHeader>
               CREATE A BOARD
-              {/* <Button
+              {hasTemplates && <Button
                 onClick={() => {
-                  console.log('view all templates')
+                  setTabValue('Templates');
                 }}
                 size='small'
                 variant='ghost'
               >
                 View all templates
-              </Button> */}
+              </Button>}
             </LayoutElements.BlockHeader>
             <LayoutElements.VStack
               style={{
                 gap: '8px',
               }}>
-              {generators.result.map(template => (
-                <DashboardTemplate
-                  key={template.op_name}
-                  title={template.display_name}
-                  subtitle={template.description}
-                  onClick={() => {
-                    setIsGenerating(true);
-                    makeBoardFromNode(
-                      template.op_name,
-                      refinedExpression.result as any,
-                      newDashExpr => {
-                        navigateToExpression(newDashExpr);
-                        setIsGenerating(false);
-                      }
-                    );
-                  }}
-                />
-              ))}
+              {recommendedTemplateInfo.op_name !== SEED_BOARD_OP_NAME &&
```

Comment:
replaces the list of all templates with a recommended template, and the seed board template